release/airflow/modify_values.sh

```diff
@@ -26,30 +26,32 @@ function fix_values_yaml_worker() {
   tarball_name="$4"
   local gcs_folder_path
   gcs_folder_path="$5"
+  local hub
+  hub="$6"
 
   gsutil -q cp "${gcs_folder_path}/${tarball_name}" .
   eval    "$unzip_cmd"     "${tarball_name}"
   rm                       "${tarball_name}"
 
-  sed -i "s|hub: gcr.io/istio-release|hub: ${HUB}|g" ./"${folder_name}"/install/kubernetes/helm/istio*/values.yaml
-  sed -i "s|tag: master-latest-daily|tag: ${TAG}|g"  ./"${folder_name}"/install/kubernetes/helm/istio*/values.yaml
+  sed -i "s|hub: gcr.io/istio-release|hub: ${hub}|g" ./"${folder_name}"/install/kubernetes/helm/istio*/values.yaml
+  sed -i "s|tag: .*-latest-daily|tag: ${TAG}|g"  ./"${folder_name}"/install/kubernetes/helm/istio*/values.yaml
```

Comment:
Should we use ${branch} instead of .*?

Reply:
no, that makes it brittle and subject to changes in the master/release branch. Since it is being replaced with the correct tag this is a better fix.